docs/README.md

```diff
@@ -19,7 +19,133 @@ tbd
 tbd
 
 ## Observability Integrations
-tbd
+
+### Scraping metrics using the OpenShift monitoring stack
+The easiest way to get started with production-grade metrics collection is to use OpenShift's user-workload monitoring stack. The following steps assume that you installed Istio into the `istio-system` namespace.
+
+*Prerequisites*
+* User Workload monitoring is [enabled](https://docs.openshift.com/container-platform/latest/observability/monitoring/enabling-monitoring-for-user-defined-projects.html)
+
+*Steps*
+1. Create a ServiceMonitor for istiod.
+
+    ```yaml
+    apiVersion: monitoring.coreos.com/v1
+    kind: ServiceMonitor
+    metadata:
+      name: istiod-monitor
+      namespace: istio-system 
+    spec:
+      targetLabels:
+      - app
+      selector:
+        matchLabels:
+          istio: pilot
+      endpoints:
+      - port: http-monitoring
+        interval: 30s
+    ```
+1. Create a PodMonitor to scrape metrics from the istio-proxy containers. Note that *this resource has to be created in all namespaces where you are running sidecars*.
```

Comment:
Instead of creating the `podMonitor` in all namespaces where sidecars are injected, there seems to be an option to specify the namespaces using [spec.namespaceSelector](https://docs.openshift.com/container-platform/4.13/rest_api/monitoring_apis/podmonitor-monitoring-coreos-com-v1.html#spec-namespaceselector)

Reply:
Okay, I see that [OCP monitoring](https://docs.openshift.com/container-platform/4.13/service_mesh/v2x/ossm-observability.html) ignores the namespaceSelector, please ignore this comment.

```
	Since OpenShift Container Platform monitoring ignores the namespaceSelector spec in ServiceMonitor and PodMonitor objects, you must apply the PodMonitor object in all mesh namespaces, including the control plane namespace.
```